rules/null_comparison_toggle.go

```diff
@@ -0,0 +1,159 @@
+package rules
+
+import (
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/terraform-linters/tflint-plugin-sdk/hclext"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+)
+
+var nullComparisonToggleBodySchema = &hclext.BodySchema{
+	Blocks: []hclext.BlockSchema{
+		{
+			Type:       "variable",
+			LabelNames: []string{"name"},
+			Body: &hclext.BodySchema{
+				Attributes: []hclext.AttributeSchema{
+					{Name: "type"},
+				},
+			},
+		},
+		{
+			Type:       "resource",
+			LabelNames: []string{"type", "name"},
+			Body: &hclext.BodySchema{
+				Attributes: []hclext.AttributeSchema{
+					{
+						Name: "count",
+					},
+				},
+			},
+		},
+	},
+}
+
+var _ tflint.Rule = new(NullComparisonToggleRule)
+
+type NullComparisonToggleRule struct {
+	tflint.DefaultRule
+}
+
+func NewNullComparisonToggleRule() *NullComparisonToggleRule {
+	return &NullComparisonToggleRule{}
+}
+
+func (t *NullComparisonToggleRule) Name() string {
+	return "null_comparison_toggle_tfnfr11"
+}
+
+func (t *NullComparisonToggleRule) Link() string {
+	return "https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tfnfr11---category-code-style---null-comparison-toggle"
```

Comment:
There is no mention of the `local` workaround in the spec here - do we need to update this as well?